src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -3,15 +3,15 @@
 # SPDX-License-Identifier: Apache-2.0
 
 """Factory functions for ops added to openvino opset13."""
-from functools import partial
+from functools import partial, singledispatch
 from typing import Literal, Optional, Union
 import logging
 
 import numpy as np
 
 log = logging.getLogger(__name__)
 
-from openvino.runtime import Node, Shape, Type, Output
+from openvino.runtime import Node, Shape, Type, Output, Tensor
```

Comment:
imports can be sorted as
```suggestion
from openvino.runtime import Node, Shape, Tensor, Type, Output
```